test/organizations/data/OrganizationCompletionEntity.js

```diff
@@ -0,0 +1,233 @@
+export const testData = {
```

Comment:
Suggect renaming file to CompletionTracking.js, see explanation in similar comments above.